lte/gateway/python/magma/pipelined/gw_mac_address.py

```diff
@@ -67,13 +67,14 @@ def get_iface_by_ip4(target_ip4: str) -> Optional[str]:
 
 
 def get_mac_by_ip6(gw_ip: str) -> str:
-    for iface, ip6 in get_ifaces_by_ip6(gw_ip):
+    for iface, _ in get_ifaces_by_ip6(gw_ip):
         # Refresh the ip neighbor table
-        if subprocess.run(["ping", "-c", "1", gw_ip]).returncode != 0:
+        if subprocess.run(["ping", "-c", "1", gw_ip], check=True).returncode != 0:
```

Comment:
Why set `check=True` here? I think this raises an exception for non-zero returncode and the logic would have to change.

Reply:
Thank you for finding this!
Imho https://pylint.pycqa.org/en/latest/user_guide/messages/warning/subprocess-run-check.html states that `check` should be set explicitly and uses `check=True` in the example. But yeah, my fault for not making sure that `False` is the default.

Changed to default `check=False`.